docs/tutorials/embedding/MASE.ipynb

```diff
@@ -0,0 +1,374 @@
+   "source": [
+    "In order to demonstrate how to use the MultipleASE class, we will use a simple example: two 2-block stochastic block models (SBMs) with different block probabilities.  Indeed, we will use the same example as in the omnibus [tutorial](https://graspy.neurodata.io/tutorials/embedding/omnibus) to facilitate direct comparison. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulate two different graphs using stochastic block models (SBM)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We sample 2-block SBMs (undirected, no self-loops) with 50 vertices, each block containing 25 vertices ($n = [25, 25]$), and the following block probabilities:\n",
+    "\n",
+    "\\begin{align*}\n",
+    "P_1 = \n",
+    "\\begin{bmatrix}0.3 & 0.1\\\\\n",
+    "0.1 & 0.7\n",
+    "\\end{bmatrix},~\n",
+    "P_2 = \\begin{bmatrix}0.3 & 0.1\\\\\n",
+    "0.1 & 0.3\n",
+    "\\end{bmatrix}\n",
+    "\\end{align*}\n",
+    "\n",
+    "The only difference between the two are the block probability for the second block. We sample $G_1 \\sim \\text{SBM}(n, P_1)$ and $G_2 \\sim \\text{SBM}(n, P_2)$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspy.simulations import sbm\n",
+    "\n",
+    "n = [25, 25]\n",
+    "P1 = [[0.3, 0.1],\n",
+    "      [0.1, 0.7]]\n",
+    "P2 = [[0.3, 0.1],\n",
+    "      [0.1, 0.3]]\n",
+    "\n",
+    "np.random.seed(8)\n",
+    "G1 = sbm(n, P1)\n",
+    "G2 = sbm(n, P2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize the graphs using heatmap\n",
+    "\n",
+    "We visualize the sampled graphs using the ``heatmap()`` function. ``heatmap()`` will plot the adjacency matrix, where the colors represent the weight of the edge. In this case, we have binary graphs so the values will be either 0 or 1. \n",
+    "\n",
+    "There is clear block structure to the graphs, and we see that the second, lower right, block of $G_1$ has more edges than that of $G_2$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspy.plot import heatmap\n",
+    "\n",
+    "heatmap(G1, figsize=(7,7), title=\"Visualization of Graph 1\")\n",
+    "heatmap(G2, figsize=(7,7), title=\"Visualization of Graph 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Embed the two graphs using MASE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "MASE fits a sample of graphs to the common-subspace independent edge (COSIE) model for multiple graphs. The COSIE model assumes that each graph is generated from a common invariant subspace, $V$, but with potentially different connection probabilities between each vertex, given by score matrices $R^{(i)}$."
```

Comment:
[i happened to have never used mase previously, but] i'm really not new to graphstat, and despite that, i simply could not parse this sentence. what does it mean to be generated from a common invariant subspace? do you think you could use try to explain in simpler terms, because your audience is likely not subject experts.

Reply:
in addition to having simpler explanation, i think i'd appreciate more explanation. maybe a very brief summary of what this embedding is math-wise.